src/main/java/org/fundacionjala/coding/fizzbuzz/Kata.java

```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.coding.fizzbuzz;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Write a program that prints the numbers from 1 to 100. But for multiples of three print Fizz instead
+ * of the number and for the multiples of five print Buzz. For numbers which are multiples of both three
+ * and five print FizzBuzz.
+ */
+public final class Kata {
+    /**
+     * Empty constructor.
+     */
+    private Kata() {
+    }
+
+    /**
+     * @param number quantity of print numbers
+     * @return The whole String
+     */
+    public static String fizzBuzz(final int number) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 11 to the 10 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)